### ISSUE TITLE
Implement ipmitool for Baremetal Openshift Node Scenarios

### ISSUE THREAD

**Issue by smalleni** [cloud-bulldozer/kraken#57]
Currently it looks like we support AWS and GCP for node scenarios like node power off. power on and reboot.

If we have an ipmitool based library for the node scenarios, we can operator against baremetal OpenShift.

Some raw calls for power on/off
```
ipmitool -I lanplus -H <host>  -U <user> -P  <password> chassis power off
ipmitool -I lanplus -H <host>  -U <user> -P  <password> chassis power on
```

Keep in mind, if a node is powered off already, the power off command will return an error, so you need to check the power status first.

https://pypi.org/project/python-ipmi/ is a library on which you can base your module.

Also, the mapping between node name and baremetal host FQDN is not straightforward on baremetal installs.
```[kni@e16-h18-b03-fc640 ~]$ oc project openshift-machine-api
Now using project "openshift-machine-api" on server "https://api.test155.myocp4.com:6443".
[kni@e16-h18-b03-fc640 ~]$ oc get bmh
NAME              STATUS   PROVISIONING STATUS      CONSUMER                       BMC                                                          HARDWARE PROFILE   ONLINE   ERROR
master-0          OK       externally provisioned   test155-77wr8-master-0         ipmi://mgmt-e16-h18-b04-fc640.rdu2.scalelab.redhat.com:623                      true     
master-1          OK       externally provisioned   test155-77wr8-master-1         ipmi://mgmt-e16-h20-b02-fc640.rdu2.scalelab.redhat.com:623                      true     
master-2          OK       externally provisioned   test155-77wr8-master-2         ipmi://mgmt-e16-h20-b04-fc640.rdu2.scalelab.redhat.com:623                      true     
storage001-r640   OK       provisioned              test155-77wr8-worker-0-p62h6   ipmi://mgmt-f04-h10-000-r640.rdu2.scalelab.redhat.com        unknown            true     
storage002-r640   OK       provisioned              test155-77wr8-worker-0-4wwqz   ipmi://mgmt-f04-h11-000-r640.rdu2.scalelab.redhat.com        unknown            true     
storage003-r640   OK       provisioned              test155-77wr8-worker-0-rb6dt   ipmi://mgmt-f04-h12-000-r640.rdu2.scalelab.redhat.com        unknown            true     
storage004-r640   OK       provisioned              test155-77wr8-worker-0-7frns   ipmi://mgmt-f04-h13-000-r640.rdu2.scalelab.redhat.com        unknown            true     
storage005-r640   OK       provisioned              test155-77wr8-worker-0-bnnls   ipmi://mgmt-f04-h33-000-r640.rdu2.scalelab.redhat.com        unknown            true     
worker-0          OK       provisioned              test155-77wr8-worker-0-jjrq6   ipmi://mgmt-e16-h24-b01-fc640.rdu2.scalelab.redhat.com:623   unknown            true     
worker-1          OK       provisioned              test155-77wr8-worker-0-jbnsq   ipmi://mgmt-e16-h24-b02-fc640.rdu2.scalelab.redhat.com:623   unknown            true     
```
is how you get the mapping between node name and management IP FQDN.

At the minimum we should support
node_start_scenario
node_stop_scenario
node_stop_start_scenario
node_reboot_scenario
in https://github.com/openshift-scale/kraken/blob/master/docs/node_scenarios.md

**Comment by paigerube14** [comment]
Jared has an open PR for this: https://github.com/cloud-bulldozer/kraken/pull/74

**Comment by chaitanyaenr** [comment]
Fixed by #74. Thanks to @jaredoconnell!
